src/sgame/sg_bot.cpp

```diff
@@ -858,7 +858,11 @@ std::string G_BotToString( gentity_t *bot )
 	{
 		return "";
 	}
-	return Str::Format( "^*%s^*: %s [s=%d b=%s g=%s]",
-			bot->client->pers.netname, BG_TeamName( G_Team( bot ) ), bot->botMind->botSkill.level,
-			bot->botMind->behaviorTree->name, BotGoalToString( bot ) );
+	return Str::Format( "^*%s^*: %s [s=%d ss=\"%s\" b=%s g=%s]",
```

Comment:
This is probably going to have issues with the message being too big for one packet. Not sure what the exact limit is bug if you had 20 bots with 20 skills each and a skill is on average 15 characters long then you have 6000 bytes of skills

Reply:
I guess since `G_admin_listbots` uses `ADMBP_Begin`, that's a problem. I don't know what to do about it though.

Reply:
And huh, to do that, is it enough to stop using `ADMBP_Begin` and just send the data using ADMP for each bot in listbots?

Reply:
So it was a lot of pain figuring that out, but I've got a version that doesn't overflow now. And also now I hate `ADMP`.